python/whylogs/migration/converters.py

```diff
@@ -21,8 +22,83 @@
     MaxIntegralComponent,
     MinIntegralComponent,
 )
-from whylogs.core.proto.v0 import ColumnMessageV0, DatasetProfileMessageV0, InferredType
+from whylogs.core.metrics.metrics import CardinalityMetric
+from whylogs.core.proto import SegmentTag
+from whylogs.core.proto.v0 import (
+    ColumnMessageV0,
+    CountersV0,
+    DatasetProfileMessageV0,
+    DatasetPropertiesV0,
+    FrequentItemsSketchMessageV0,
+    HllSketchMessageV0,
+    InferredType,
+    NumbersMessageV0,
+    SchemaMessageV0,
+    VarianceMessage,
+)
+from whylogs.core.segment import Segment
+from whylogs.core.segmentation_partition import SegmentationPartition
 from whylogs.core.utils import read_delimited_protobuf
+from whylogs.core.utils.timestamp_calculations import to_utc_milliseconds
+
+EMPTY_FI_SKETCH: bytes = ds.frequent_strings_sketch(10).serialize()
+EMPTY_HLL: bytes = ds.hll_sketch(12).serialize_compact()
+EMPTY_KLL: bytes = ds.kll_doubles_sketch(k=128).serialize()
+_empty_theta_union = ds.theta_union()
+_empty_theta_union.update(ds.update_theta_sketch())
+EMPTY_THETA: bytes = _empty_theta_union.get_result().serialize()
+
+logger = getLogger(__name__)
+
+PARTITION_ID = "segp_id"
+PARTITION_NAME = "segp_name"
+PARTITION_HAS_FILTER = "segp_filter"
+SEGMENT_ON_COLUMN = "segp_col"
+SEGMENT_ON_COLUMNS = "segp_cols"
+_TAG_PREFIX = "whylogs.tag."
+
+
+def _generate_segment_tags_metadata(
```

Comment:
Gosh, what a signature. Although true, and the return is fairly readable.. if we're returning 3 related but different dictionaries and lists, why not just go ahead and make it a DataClass?

Reply:
I don't like the signature either, I factored this into a method to reuse the logic, we can refactor better later, but deferring the follow up here if acceptable?

Reply:
Yeah I think it's fine to defer to something a little cleaner later